cdap-proto/src/main/java/co/cask/cdap/proto/profile/ProfileMeta.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.profile;
+
+import co.cask.cdap.runtime.spi.profile.ProfileStatus;
+
+import java.util.Objects;
+
+/**
+ * Profile meta data which stores the information about the profile and the profile status
+ */
+public class ProfileMeta {
```

Comment:
Doesn't this break the REST response? Believe status should just be a field in Profile instead of having this extra layer

Reply:
removed this class and add status to Profile